modules/prometheus/multi_metadata.yaml

```diff
@@ -264,3 +264,7108 @@ modules:
 #      data_collection:
 #        <<: *data_collection
 #        metrics_description: DESCRIPTION
+  - <<: *module: 
+    meta:
+      <<: *meta: 
+      most_popular: false
+      monitored_instance:
+        name: A10-Networks
+        link: https://github.com/a10networks/PrometheusExporter
+        icon_filename: a10-networks.png
+        categories:
+        - data-collection.networking-stack-and-network-interfaces
+      keywords:
+      - network monitoring
+      - network performance
+      - traffic analysis
+    overview:
+      <<: *overview: 
+      data_collection:
+        <<: *data_collection:
+        metrics_description: Monitor your A10-Networks metrics.
```

Comment:
I think we need to mention the exporter somewhere too. I think it needs to be in:

- `method_description` and

```
It collects metrics by periodically sending HTTP requests to [X exporter](LINK) and parsing the response.
```

- `setup.prerequisites.list` (Install X exporter)